responses/sv/HassTurnOn.yaml

```diff
@@ -22,16 +22,16 @@ responses:
           "lawn_mower": "gräsklipparen"
         } %}
         Slog på {{ translations.get(state.domain) }}
-      lights: "Tände {{ slots.name }}"
+      light: "Tände {{ slots.name }}"
       lights_area: "Tände lamporna i {{ slots.area }}"
       lights_all: "Tände alla lamporna"
       lights_floor: "Tände lamporna på {{ slots.floor }}"
       fans_area: "Startade fläktarna i {{ slots.area }}"
       fans_all: "Slog på alla fläktar"
-      cover: "Öppnade {{ slots.name }}"
+      cover: "Öppnade"
       cover_area: "Öppnade i {{ slots.area }}"
-      cover_device_class: "Öppnade {{ slots.device_class }}"
```

Comment:
This is not the correct change, we don't want the response for a device_class to not include the class.
The intention it that this will say
`Öppnande garaget` if a command was issued for a device that will open that device_class.